app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPane.scala

```diff
@@ -161,9 +164,107 @@ class DLCPane(glassPane: VBox) {
     spacing = 10
   }
 
+  private val tableView = {
+    val eventIdCol = new TableColumn[DLCStatus, String] {
+      text = "Event ID"
+      prefWidth = 150
+      cellValueFactory = { status =>
+        new StringProperty(status, "Event ID", status.value.eventId.hex)
+      }
+    }
+
+    val statusCol = new TableColumn[DLCStatus, String] {
+      text = "Status"
+      prefWidth = 150
+      cellValueFactory = { status =>
+        new StringProperty(status,
+                           "Status",
+                           DLCStatus.statusString(status.value))
+      }
+    }
+
+    val initiatorCol = new TableColumn[DLCStatus, String] {
+      text = "Initiator"
+      prefWidth = 80
+      cellValueFactory = { status =>
+        val str = if (status.value.isInitiator) {
+          "YES"
+        } else {
+          "NO"
```

Comment:
nit: I prefer `Yes` and `No` but do what you want

Reply:
I'll probably leave it all caps for now because that's what I did with status as well, but open to changing this in the future of course